simulator/autoware_vehicle_door_simulator/src/dummy_doors.cpp

```diff
@@ -16,10 +16,10 @@
 
 #include <memory>
 
-namespace vehicle_door_simulator
+namespace autoware::simulator::vehicle_door_simulator
 {
 
-DummyDoors::DummyDoors() : Node("dummy_doors")
+DummyDoors::DummyDoors() : Node("autoware_vehicle_door_simulator_node")
```

Comment:
Node name do not need the autoware prefix

Reply:
Thank you, I'll also apply the fix for the other PRs.

Reply:
Fixed by [this commit](https://github.com/autowarefoundation/autoware.universe/pull/9997/commits/1b0c181ae9b874cf64aded19682ebf2703bb2502)